libs/database/readme.md

```diff
@@ -0,0 +1,17 @@
+# Spin up local database
+
+1. Pull the database image
+   ```
+   docker pull postgres:15-alpine
+   ```
+2. Start the database
+   ```
+   docker run \
+     -p 5432:5432 \
+     -e POSTGRES_PASSWORD=password \
+     postgres:15-alpine
+   ```
+3. Push the migrations to the database
+   ```
+   DATABASE_URL=postgres://postgres:password@localhost:5432 yarn db:migrate-prod
```

Comment:
Maybe just me, but a bit confused.
Not sure if just naming or a small "bug".

Spin up local database -> yarn db:migrate-prod
Should this be yarn db:migrate-dev?

Reply:
It is correct 😅 But I agree that the naming of the scripts might be a bit misleading. I'll rename it to something closer to the actual command it runs;

The old/current scripts right now for reference:
```
    "db:migrate-dev": "prisma migrate dev",
    "db:migrate-prod": "prisma migrate deploy",
```

The `prisma migrate dev` will generate new migration files and push changes – the `prisma migrate deploy` will only look at the migration files and push those changes.

Reply:
Suggestion:

```
    "db:migrate-generate": "prisma migrate dev",
    "db:migrate": "prisma migrate deploy",
```